.github/workflows/testing.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
       github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name != github.repository
     strategy:
       matrix:
-        tarantool-version: ['1.10', '2.2', '2.3', '2.4', '2.5', '2.6', '2.7', '2.8']
+        tarantool-version: ['1.10', '2.2', '2.4', '2.5', '2.6', '2.7']
```

Comment:
Any comment regarding this?

Reply:
It was a temporary commit and now reverted.